app.js

```diff
@@ -68,4 +68,62 @@ let sum1 = 0
 for (let i = 11; i <= 30; i += 2) {
 	sum1 += i;
 }
-console.log(sum1);
\ No newline at end of file
+console.log(sum1);
+
+console.log("==============Challenge 8=================");
+// Coding challenge #8: Create a function that will convert from Celsius to Fahrenheit
+
+
+function celciusToFahrenheit(celcius) {
+	return (celcius * 9 / 5) + 32;
+}
+
+console.log("==============Challenge 9=================");
+// Coding challenge #9: Create a function that will convert from Fahrenheit to Celsius
+
+
+function fahrenheitToCelcius(fahrenheit) {
+	return (fahrenheit - 32) * 5 / 9;
+}
+
+console.log("==============Challenge 10=================");
+
+// Coding challenge #10: Calculate the sum of numbers in an array of numbers
+
+function sumOfArray(arr) {
+	let sum = 0
+	for (i of arr) {
+		sum += i;
+	};
+	return sum
+}
+console.log("==============Challenge 11=================");
+
+// Coding challenge #11: Calculate the average of the numbers in an array of numbers
+
+
+function averageOfArray(arr) {
+	let sum = 0
+
+	for (i of arr) {
+		sum += i;
+	};
+	return sum / arr.length;
+}
+
+console.log("==============Challenge 12=================");
+// Coding challenge #12: Create a function that receives an array of numbers and returns an array containing only the positive numbers
+
+
+function positivesOfArray(arr) {
+	let positives = []
+
+	for (i of arr) {
+		if (i >= 0) {
+			positives.push(i);
+		}
+	}
+	return positives;
+}
```

Comment:
💯 .

Just change the `i` to something better named (such as `value`) and you're good to go.